src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -796,21 +816,59 @@ protected void CheckAssigned(Symbol symbol, CSharpSyntaxNode node)
                         ReportUnassigned(symbol, node);
```

Comment:
This actually calls a different, virtual method. Inheriting passes use this path to do something different for ReportUnassigned.